plugins/modules/aci_domain_to_vlan_pool.py

```diff
@@ -330,21 +330,22 @@ def main():
             module_object=domain_mo,
             target_filter={"name": domain},
         ),
-        child_classes=["infraRsVlanNs"],
+        subclass_1=dict(
+            aci_class="infraRsVlanNs",
+            aci_rn="rsvlanNs",
+            target_filter={"dn": "{0}/rsvlanNs".format(domain_mo)},
```

Comment:
I think the target_filter should be the tDn with its value

Reply:
also missing then module_object definition

Reply:
Hi. the tDn on the class is the actual vlan_pool name being bound, we could also filter on that. There can only be 1 binding per rsvlanNs class, so the filter is always going to bring the expected binding:
```
    "imdata": [
        {
            "infraRsVlanNs": {
                "attributes": {
                    "dn": "uni/phys-phydom_aib/rsvlanNs",
                    "tDn": "uni/infra/vlanns-[vpool_static]-static",
                }
````
I've changed to tDn and seems to be working as expected. Also added the module_object.
Working on tests soon.